settings.gradle

```diff
@@ -4,6 +4,7 @@ include ':sample',
         ':rx-extensions',
         ':scheduler',
         ':dfd',
+        ':dfd-wrapper',
```

Comment:
please also prep this for publishing, it seems some files with versions and artifact names is missing

Reply:
I've added `gradle.properties` like all the other modules have. Is that enough?

Reply:
we also need new git actions workflow, should be easy to copypaste from dod_publish_maven_action.yml

Reply:
Done 😁